Add modified nation descriptions

Ouaz opened the pull request:
New nation descriptions, reworded and shortened.

Requires to be reviewed/corrected/improved by a native english speaker.

This PR serves as a draft, someone can copy the file, improve/correct it, and create another PR if needed, or post suggestions/corrections here directly.

Vanilla nations.json file will be added later, once final descriptions will be ready (to not have to review/modify two files).

EDIT: I made a big historical mistake in one of the description. Let's see if someone notices it ^^

----------------------------------------------

**EDIT December 8th, 2023: added the modified descriptions in vanilla nations.json.**

**The 2 files are ready to merge, but no native english speaker has reviewed it. If some improvements need to be made, translators will have to redo the translation. ;)**

ravignir commented:
Did you mean Alexander's description?

Caballero-Arepa commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -105,8 +105,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O great Ramesses, Pharaoh of Egypt, bright as the Sun and mighty as the Nile! Lord of all living things, you lead the greatest of all civilizations, which has gifted the world with writing, advanced mathematics, sculpture, and architecture. The egyptian culture and monuments amaze the world since thousands of years, keeping Egypt to the pinacle of humanity achievement. ...
```
I disagree with the use of superioty over the rest. Yeah, like I say that they should be more modest.

yairm210 replied:
That is canonically how Egypt thought of themselves :)
The last line is missing capitalization for Egyptian, and needs general rewording:
"Egyptian monuments and culture will continue to amaze the world for thousands of years" without the end bit

Caballero-Arepa commented:
All in all it pretty great!

Very very good job. I even dare to say that these are way better than the original ones.

I love them a lot.

Ouaz commented:
> I disagree with the use of superioty over the rest. Yeah, like I say that they should be more modest.

I don't know... the emphasis is what makes the fun ot it, otherwise it would be just a boring text about the nation history (and the corresponding leader).

I see it more like a "courtier" speaking to the nation leader, hence the exaggeration and the bragging "(_you're the best, you're so perfect, our nation rules_" :sunglasses: )
Every nation is treated equally anyway and it's in line with how nation leaders speak to the player in-game.

> Very very good job. I even dare to say that these are way better than the original ones.
> I love them a lot.

I just took the original ones, reworded/shortened them, and added even more emphasis. ^^
And also added some light "innuendos" for some nations here and there. :smirk:

estorski commented:
Shortening is good, but OMG, that's gonna be another nightmare to translate so many intros to so many languages 😭

Ouaz commented:
> Shortening is good, but OMG, that's gonna be another nightmare to translate so many intros to so many languages

It depends. If the previous english intros were already translated in your language, you can take these translations as a base, and compare what have been modified: Intros are shortened, some words are replaced, but there's still a "global similarity" with the previous ones.

For my part, I'll do that for french translation, rather than translating again the intros from scratch.

Caballero-Arepa commented:
> Shortening is good, but OMG, that's gonna be another nightmare to translate so many intros to so many languages 😭

Hah, nah its not.

Just translate one per day if you get bored xD

yairm210 commented:
@Ouaz Are you going to un-draft this / keep working / close?

Ouaz commented:
Well, I was waiting an english native speaker to fully review that, but it seems nobody is motivated ^^
I can un-draft it, and also add the vanilla json file too, if you want.

alexban011 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
lgtm ...

alexban011 commented on android/assets/jsons/Civ V - Vanilla/Nations.json:
again... lets get this merged.

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -14,8 +14,8 @@
 
-		"startIntroPart1": "",
-		"startIntroPart2": "",
+		"startIntroPart1": "May the gods bless you, great Nebuchadnezzar, ruler of ancient Babylon! Built 5000 years ago, Babylon emerged as an empire in 1800 BC only, thanks to godlike Hammurabi, The Giver of Law. Fallen under the Kassites assaults then the Assyrian domination, Babylon reborned from its ashes, gaining back its independance, and became the beacon for Arts and Learning in the ancient wo ...
```
Remove 'only'
Fallen -> Falling
Reborned -> was reborn

Instead of 'sadly didn't last long' put 'quickly fell'

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -45,8 +45,8 @@
 
-		"startIntroPart1": "",
-		"startIntroPart2": "",
+		"startIntroPart1": "Great King Alexander, leader of the mighty Greek nation, you are truly blessed by the gods. By your astonishing conquests, you showed once again that union leads to greatness. In the past, quarrelsome city-states such as Athens, Sparta, Thebes, found a way to join their forces, allowing Greece to defeat the mighty Persia in the 5th century BC, both on land and sea. Democracy ...
```
The victory open Persia seems unrelated
Maybe "From Europe to Africa to India, you built an empire unmatched by any before it."

Ouaz replied:
> The victory open Persia seems unrelated
I want to say that Alexander followed the example of Athens, Sparta, Thebes who united their forces to defend against Persia invasion, and defeat them (480 BC).
Alexander did the same in 334 BC: he founded a coalition (Macedonia and Greece cities) to invade Persia, and strike it down.

I rewrite it, keeping your suggestion.

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -76,8 +76,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Heaven itself bends before your beauty, Empress Wu Zetian, ruler of eternal China! You lead one of the oldest and greatest civilization of humanity. In the old ages, chinese people already mastered a level of culture and science that many other civilizations will reach much later in history. China brought so many contributions to human knowledge - the printing press, g ...
```
Point here is repeated. Instead:

"In your time, China stood at the forefront of science and technology, bringing the world such contributions as gunpowder, the printing press, and the world of Confucius."

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -137,8 +137,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Long live to Your Highness, Queen Elizabeth Gloriana, leader and protector of the glorious England! Through history, enduring many invasions, England strengthened its will, playing smartly with diplomacy, cunning, and military power, to eventually rose as one of the most dominant Empire the world has ever known. Their matchless navy, brilliant scientists and artists, a ...
```
Empire - Empires

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -164,8 +164,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Vive Napoleon I, First Consul and Emperor of France! After the glorious Revolution which suppressed the centuries-old monarchy, the French nation stood at the heart of Europe as a Republic, and intended to export its ideals to the entire European continent, either by diplomacy or war. The french capital Paris was more than ever the world center of Culture, Arts, and Le ...
```
French capital should e capitalized (pun)

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -193,8 +193,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Hail to Imperial Majesty Catherine, Empress of all the Russias. Ruling the largest country in the world, from the Pacific Ocean in the east to the Baltic Sea in the west, your people endured wars, droughts, and many disasters, imbuing the Russian society with heroism and tenacity, and producing artists and scientists among the best in the world. Russia is a powerful em ...
```
Of all Russians

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -193,8 +193,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Hail to Imperial Majesty Catherine, Empress of all the Russias. Ruling the largest country in the world, from the Pacific Ocean in the east to the Baltic Sea in the west, your people endured wars, droughts, and many disasters, imbuing the Russian society with heroism and tenacity, and producing artists and scientists among the best in the world. Russia is a powerful em ...
+        "startIntroPart2": "Empress Catherine, your people are eager of glory for Russia, to reborn the sacred land and enlighten the world once again. Will you achieve greatness for your country again? Can you forge an empire that lasts through the ages of history?",
```
Eager for Russian glory, to revive

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -251,8 +251,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Blessings of the Great God be upon you, O great caliph Harun al-Rashid, leader of the mighty Arabian people! The Muslim Empire, the Caliphate, born from chaos after the death of the prophet Muhammad in 632 AD, intended to apply the rule of God to all Earth. By the will of God, the caliphate reached its full power, ruling Spain, North Africa, the Middle East, Anatolia,  ...
```
The arts and sciences *were* a gift
delved deep

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -251,8 +251,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Blessings of the Great God be upon you, O great caliph Harun al-Rashid, leader of the mighty Arabian people! The Muslim Empire, the Caliphate, born from chaos after the death of the prophet Muhammad in 632 AD, intended to apply the rule of God to all Earth. By the will of God, the caliphate reached its full power, ruling Spain, North Africa, the Middle East, Anatolia,  ...
+        "startIntroPart2": "Great Caliph Harun al Rashid, all arabian people long for greatness! Arabia must be once again the land of arts and knowledge, which with God's help, will fear no enemy! Can you forge an empire that lasts through the ages of history?",
```
All Arabian people

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -308,8 +308,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Lord Oda Nobunaga, ruler of Japan, the land of the Rising Sun is at your command! Through history, japanese people lived and died with honor, proud of their rich culture of arts and letters. Years of bloody civil war, invasions of foreign powers, wealth but also great poverty, have never altered the strong will of the Rising Sun island, as the Bushido spirit has imbued ...
```
Japanese - capitalized

With which only few nations can compete

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -308,8 +308,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Lord Oda Nobunaga, ruler of Japan, the land of the Rising Sun is at your command! Through history, japanese people lived and died with honor, proud of their rich culture of arts and letters. Years of bloody civil war, invasions of foreign powers, wealth but also great poverty, have never altered the strong will of the Rising Sun island, as the Bushido spirit has imbued ...
+        "startIntroPart2": "O great daimyo, will you take on your hands the Japan destiny? Will honor and glory infuse again on your people like cherry flowers blossom in the wind? Can you forge an empire that lasts through the ages of history?",
```
Again infuse your people

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -336,8 +336,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Wise Mahatma Gandhi, great leader of India, we honor you! India is one of the oldest countries in the world with 10,000 years of history, built on spirit and religion, as three of the world's great religions - Hinduism, Buddhism and Jainism - appeared on this sacred land. Every aspect of India society reveres art, music and color, showing great wealth but also grinding ...
```
Have led

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -362,8 +362,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "All hail to Otto von Bismarck, first chancellor of Germany and its empire! During the ancient era, Germany was a land fragmented in several tribes but already showed great promises as the Roman Empire itself struggled to bend the Germans to its will. Germany, as one nation, was slowly built on the ruins of the Holy Roman Empire, and waited 1871 to be finally unified af ...
+        "startIntroPart2": "Great Prince Bismarck, glory and power are at the heart of the German nation. Your people are strong, and lend you their iron will, to lead Germany once again. Can you forge an empire that lasts through the ages of history?",
```
Was he a prince...?

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -394,8 +394,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O Great Emperor Suleiman, may the God bless you! You are revered for your power, wealth and generosity, and you truly deserves to be called the 'Magnificent!' The Ottoman Empire was born in Bithynia, starting as a small country in Eastern Anatolia in 12th century, then expanded to west into Anatolia. Over the next century, your people vanquished the empire of Byzantium ...
```
And knowledge

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -423,8 +423,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Greetings to you, King Sejong the Great, heir of the Choson Dynasty! Prosperity and benevolence towards the common man are the principles of you reign, and made of you the most beloved korean king. You always aimed to provide a fair and equal society for all your people, and also to favorize knowledge and technological discoveries. Your wisdom and intelligence imbued a ...
```
Of your reign
Korean capitalized
And also to favor

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -423,8 +423,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Greetings to you, King Sejong the Great, heir of the Choson Dynasty! Prosperity and benevolence towards the common man are the principles of you reign, and made of you the most beloved korean king. You always aimed to provide a fair and equal society for all your people, and also to favorize knowledge and technological discoveries. Your wisdom and intelligence imbued a ...
+        "startIntroPart2": "Wise Sejong, will you guide your people once again? Will harmony and knowledge be again the social bases of Korea, to reach new heights of achievement? Can you forge an empire that lasts through the ages of history?",
```
Social basis

yairm210 replied:
Reaching new heights

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -452,8 +452,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Greetings, noble Hiawatha, leader of the mighty Iroquois nations! Your people lived near the great and holy lake Ontario since ages, on a land that will be later known as New York city. Long ago, in the antiquity, the five peoples of Seneca, Onondaga, Mohawks, Cayugas and Oneida became one nation, the Haudenosaunee, the Iroquois. Although not having a written language, ...
```
Which can be considered the first established constitution

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -452,8 +452,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Greetings, noble Hiawatha, leader of the mighty Iroquois nations! Your people lived near the great and holy lake Ontario since ages, on a land that will be later known as New York city. Long ago, in the antiquity, the five peoples of Seneca, Onondaga, Mohawks, Cayugas and Oneida became one nation, the Haudenosaunee, the Iroquois. Although not having a written language, ...
+        "startIntroPart2": "Proud Hiawatha, do you hear the complaints of you people? They seek a path through peace and war, to rebuild the Iroquois union. Will you help them, great leader? Can you forge an empire that lasts through the ages of history?",
```
Your people

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -479,8 +479,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Great King Darius of Persia, the heaven casts its light upon you! Persian people are strong and wise, as their leader. In old ages, the great Persian king Cyrus fought back the mighty Median empire and finally destroyed the Medes in 550 BC. Cunning diplomacy and military achievements helped then to conquer the wealthy Lydia and the powerful Babylon, Cyrus' son conqueri ...
```
The Persian Empire 
Brutally destroyed

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -479,8 +479,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Great King Darius of Persia, the heaven casts its light upon you! Persian people are strong and wise, as their leader. In old ages, the great Persian king Cyrus fought back the mighty Median empire and finally destroyed the Medes in 550 BC. Cunning diplomacy and military achievements helped then to conquer the wealthy Lydia and the powerful Babylon, Cyrus' son conqueri ...
+        "startIntroPart2": "King Darius, the people of Persia awaits, longing for its past glory and power! The ancient Persia must reborn, to punish its foes and to bring peace and order to the world! Can you forge an empire that lasts through the ages of history?",
```
Await

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -510,8 +510,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Aloha to Kamehameha the Great, who the heavens allowed to unite your people. O great King, under you reign, the Big Island of Hawai'i became one in 1791 AD, and many other islands joined your vision of union in 1810. As the first King of Hawai'i, law and taxation systems were stabilized, and you promoted the Mamalahoe Kawanai, an edict to protect civilian people in tim ...
```
Under your reign
after you joined your ancestors in paradise

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -538,8 +538,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O Great King Ramkhamhaeng, the Siamese people obeys you with respect and fear! The beautiful and mysterious land of Siam is an ancient country in the heart of Southeast Asia, which was besieged by many foes, plagued by bloody war and great poverty, but eventually the smart and loyal Siamese people have endured and triumphed. Siam was a part of the Khmer Empire until th ...
```
obey you

Ouaz commented:
Thanks for the reviews. I'll fix all of them ASAP. :+1: 

> Either little variations so no two are identical (e.g. only the [Inca](https://en.wikipedia.org/wiki/Prisoners_of_the_Sun) use "outshine"???) - or - rethink far far away from the original so it's unassailable???

About that... What's your opinion?
I can reword `"Can you forge an empire that lasts through the ages of history?"` for each nation, but you'll have to review them. ^^

yairm210 commented:
Having the same end line for each nation gives it a sense of unity, it's good :)

Ouaz commented:
> Having the same end line for each nation gives it a sense of unity, it's good :)

Yes, but thinking of it, maybe SomeTroglodyte made a point.
Rather than paraphrasing "_Can you build a civilization that will stand the test of time?_" with "`Can you forge an empire that lasts through the ages of history?`", maybe we can use a totally different sentence, that defines Unciv?

- Can you make your empire prevail once again?
- Can you accomplish the impossible once again?
- Can you make your civilization great again? (lolz)

Some sort of a good "motto" sentence.

Ouaz commented:
Yes, 34+9 variations is a bit... too much. :sweat_smile:

Choose only one pills.

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
Celebrate
Had taken
They few could complete with for hundreds of years

yairm210 replied:
The typoer has become the typee

Ouaz commented:
Wait before merging it. I currently write again the final sentence for each nation.
To keep a sense of unity (and havea  final line more "conceptually" different than the Civ5 one) , all `StartIntroPart2` use a "bring the light to the world" idea.

For example:
- Babylon:
_"startIntroPart2": "Great Nebuchadnezzar, Babylon's glory still lives through you and your people. Will you create a spark to enlighten the world again? Will your new empire shine through the ages of history?"_

- Greece
_"startIntroPart2": "Megalexandros, son of Zeus, will you ride Bucephalas again, to gift the world with the light of the Greek culture? Will your new empire shine through the ages of history?"_

- China
_"startIntroPart2": "Great Queen, can you make China prevail again, using your divine intelligence and beauty? China stands for greatness, and all its enemies will be dazzled soon enough! Will your new empire shine through the ages of history?"_

- Egypt
_"startIntroPart2": "Mighty and eternal Ramesses, Egypt lasted for millenia, whereas other civilisations disappeared into dust after a few centuries. Amun-Ra bless you again to lead Egypt and its people on the path of light. Will your new empire shine through the ages of history?"_

Etc.

Let me know if it's a good idea or not (at least, I'll do it like that in french, it sounds better and more original imo).

yairm210 commented:
Looks incredible!
We can do this gradually if you want
Also haven't had time to go over all texts, maybe today...?

yairm210 commented:
If you're okay with merging this now, I am :)

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
Powerful Spanish military fleets
reached a state of power few could compete with for hundreds of years

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -595,8 +595,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Askia, leader of the Songhai people and greatest of all, may God watch your path towards glory! Songhai was under the rule of the mighty West African state of Mali until the middle of the 14th century. Great King Sunni Ali Ber fought for independance, and showed to all Songhai's power, by conquering territories and repelling many foes who sought to destroy the rising k ...
```
Ultimately, conquest of the wealthy cities of Timbuktu and Jeanne made Songhai
until the empire bowed down

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -622,8 +622,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O great Temuujin, universal leader of the mighty Mongol Empire! Your name alone was enough to make cities crumble, and the sound of the mighty Mongol cavalry to make flee your enemies. O Great Khan! Once Northern Asia united into a mighty people, producing one of the greatest army the world has ever known, your skillfull mounted archers conquered in only a few years, m ...
```
To make your enemies flee.
One of the greatest armies the world has even known
Skillful

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -650,8 +650,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "O divine Montezuma, your people welcomes you! May your magnificence inspire any living things! The mighty Aztecs were nomads until the 12th century, when they choose to settle in the mesa central, which will be called Mexico centuries later. Wars were frequent with other tribes to control the rich land surrounding the sacred lakes of Texcoco, Xaltocan and Zampango. In  ...
```
Your people welcome you!
Inspire all living things
Which would later be called Mexico
made the great Aztec civilization crumble

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -682,8 +682,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "King Pachacuti Sapa Inca, maker of the world and ruler of Tawantinsuyu and the Inca people, your loyal subjects greet you! In the beginning, the small state of Cusco already showed the Incans were promised for greatness. They won many wars against powerful enemies, relentlessly crushing any army that dared oppose them, and forged a mighty empire stretching from Ecuador ...
```
Not just undaunted soldiers

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -723,8 +723,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Skal, Harald Bluetooth Gormsson, son of King Gorm of the Old and Thyra Dannebod! After crushing the armies of Norway, you achieved great construction projects all across the land of mighty Denmark. Numerous Ring Fortresses were built under your reign to protect the people from foreign invasions, but also from local quarrels. In 983 AD, you successfully repelled several ...
+        "startIntroPart2": "As a true Viking, who doesn't know fear, time has come to rise up and lead your great people by the will of the Gods! Will you unleash mighty armies of Northsmen to bring despair upon your enemies? Will your enemies flee at the vision of your Drakkar fleets and the say of your very name? Can you forge an empire that lasts through the ages of history?",
```
Who knows not fear

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -753,8 +753,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Great Attila, mighty warrior and leader of the Hunnic empire, your men bow down before you. During the 5th century, you and your brother Bleda have conquered many lands, forged a mighty empire and commanded the most powerful and frightening force of antiquity. You inspired fear to the Roman Emperors themselves, who called you \"Flagellum Dei\", the \"Scourge of God\",  ...
```
Please use ' instead of "
Invaded the Eastern Roman Empire, into Gaul (they're all in Europe)

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -778,8 +778,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Greetings to the wise and brave Prince William of Orange, who freed the Netherlands and gave strength to the Dutch people. In 1568, you led a rebellion against Spanish dominion, gaining freedom for your people, and therefore starting the Eighty Years' War. Your actions were all aimed to create one of Europe's first modern republics, the Seven United Provinces. The rebe ...
+        "startIntroPart2": "Brave prince, your people asks for your wisdom to show the path to liberty again. Will you once again lead your people into modernity, long before any other civilization? Can you forge an empire that lasts through the ages of history?",
```
Ask you for your wisdom

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -804,8 +804,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "All hail Great King Gustavus Adolphus, founder of the Swedish Empire, and one of the most famous military tactician. Sweden emerged as a great power in Europe under your reign, thanks to your wisdom, both in diplomacy and warfare. As king, you decided to reform swedish laws to bring economic stability and prosperity to your people. Your allies, and also your enemies, c ...
```
Tacticians
Swedish capitalized
" To '

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -865,8 +865,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Greetings to you, revered Queen Dido, ruler of the legendary kingdom of Carthage. The great poet Virgil related your great history, how your husband Acerbas was murdered by your own brother, King Pygmalion of Tyre, to steal the treasures of your family that were rightfully yours. But you endured his pressure until you and your compatriots decided to flee and sail for n ...
```
Remove "for the country"
And laid the basis

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -835,8 +835,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Noble and pious Queen Maria Theresa, Holy Roman Empress and sovereign of Austria, the people revere your grace. After the death of your father King Charles VI, you inherited the throne of Austria during difficult economic and military times for your country. But you used your wisdom and unflinching determination to fend off your foes and resolve any obstacles to stabil ...
```
Remove "for the country"
As you gifted it

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -894,8 +894,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Magnificent Empress Theodora, beloved of Byzantium and of Rome, your people salutes you! With your beauty and intelligence, you rose up from actress and courtesan to the rank of the most powerful woman in the Roman Empire, as wife of Justinian I. In the late 520's AD, you helped your husband to engage spiritual and legal reforms, promugalting numerous laws to improve w ...
```
Your people salute you
You helped your husband engage in
To improve women's status

Ouaz commented:
I'll fix all that ASAP.
In the same time, I will add the modified final sentence as stated earlier (which will need a review)
This way, once merged, it will be once and for all (otherwise translators would have to translate everything again, each time fixes are merged afterwards).

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -894,8 +894,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Magnificent Empress Theodora, beloved of Byzantium and of Rome, your people salutes you! With your beauty and intelligence, you rose up from actress and courtesan to the rank of the most powerful woman in the Roman Empire, as wife of Justinian I. In the late 520's AD, you helped your husband to engage spiritual and legal reforms, promugalting numerous laws to improve w ...
+        "startIntroPart2": "Beautiful Empress, your people rely on your wisdom and strength. The Byzantine Empire magnificience is intact in all memories, and is ready to reborn. Can you reach again its legendary greatness? Can you forge an empire that lasts through the ages of history?",
```
```suggestion
        "startIntroPart2": "Beautiful Empress, your people rely on your wisdom and strength. The Byzantine Empire's magnificence is intact in all memories, and is ready to reborn. Can you reach again its legendary greatness? Can you forge an empire that lasts through the ages of history?",
```

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -922,8 +922,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "May glory and honor always be upon you, Warrior Queen! In a time dominated by men, you successfully claimed the throne and the power, but also valiantly defied the mighty Roman Empire. Punished and humiliated by the hand of the Roman bloody invaders, you united your people and led a terrible revolt, to seek revenge. Many legions fell under your fierce attacks and you b ...
```
```suggestion
        "startIntroPart1": "May glory and honor always be upon you, Warrior Queen! In a time dominated by men, you successfully claimed the throne and the power, but also valiantly defied the mighty Roman Empire. Punished and humiliated by the hand of the Roman bloody invaders, you united your people and led a terrible revolt, to seek revenge. Many legions fell under your fierce attacks and you burned the city of London with your anger . While the Romans still kept ownership of your land after your final defeat, your actions nonetheless showed Rome that this land could never be fully conquered, making Nero himself consider to withdraw all troops and leaving Britain forever.",
```

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -962,8 +962,8 @@
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "May God bless you, wise and righteous Emperor of Ethiopia, Haile Selassie. You are one of Ethiopia's greatest rulers and a spiritual leader of the Rastafarian movement. Your good influence on diplomacy and political cooperation are praised throughout the world. You created Ethiopia's first written constitution, and favorized democracy for years to come, your great wisd ...
```
```suggestion
        "startIntroPart1": "May God bless you, wise and righteous Emperor of Ethiopia, Haile Selassie. As one of Ethiopia's greatest rulers and a spiritual leader of the Rastafarian movement, your good influence on diplomacy and political cooperation are praised throughout the world. You created Ethiopia's first written constitution, and favorized democracy for years to come, your great wisdom making Ethiopia enter as a charter member in the United Nations. Always aiming to reform and modernize your nation, you changed Ethiopian destiny for the best.",
```

yairm210 commented on android/assets/jsons/Civ V - Gods & Kings/Nations.json:
```diff
@@ -1000,8 +1000,8 @@
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "Kneel before King Pacal the Great, son of the gods and protector of the Palenque domain! After being harassed for years by your neighboring rivals, you struck back at the enemies of your people, defeating each of them and sacrificing their leaders as a retribution to the insults made to your ancestors. Your wisdom and hard will not only restored the glory of Palenque,  ...
+        "startIntroPart2": "Absolute King, your people needs you to face the coming storm. Imbued with the will of the gods, will you restore once again the glory of your kingdom? Will your people build new monuments to show the greatness of your civilization? Can you forge an empire that lasts through the ages of history?",
```
```suggestion
        "startIntroPart2": "Absolute King, your people need you to face the coming storm. Imbued with the will of the gods, will you restore once again the glory of your kingdom? Will your people build new monuments to show the greatness of your civilization? Can you forge an empire that lasts through the ages of history?",
```

yairm210 commented:
I've made all the changes I'm likely to, if you can copy these into the Vanilla I'll be good to merge this :)
I'm sure there will be further changes but we can deal with them later ;)

yairm210 commented:
@Ouaz Tell me when you think this is ready and we'll merge it :D

Ouaz commented:
I'm good, I won't change anything more.^^

The latest commit only modified `startIntroPart2` for each nation (shortened them a bit, with a reference to "light", "brightness", etc). I _think_ the english is correct.

You can check it (roughly 34 lines, as the last one is always "_Will your new empire shine through the ages of history?_").
Or merge it as it is.

If there's some misspelling or the english is incorrect, it could be fixed afterwards (re-translate won't be a chore for translators, as there's most of the time only 2 lines in `startIntroPart2` (and they can copy their previous translation they already did it).

yairm210 commented:
I always search and replace for translated texts so no worries
